pkg/restapi/rp/operation/operations.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/trustbloc/edge-adapter/pkg/internal/common/support"
+)
+
+// API endpoints.
+const (
+	hydraLoginEndpoint                 = "/login"
```

Comment:
Are these the same endpoint as in edge-sandbox hydra login and consent. I mean does this impact sandbox ?

Reply:
yes

Reply:
@talwinder50 @fqutishat 

The design currently doesn't assume the same hydra instance. Connecting it to the same hydra instance requires additional work.